lib/matplotlib/axes/_axes.py

```diff
@@ -1084,6 +1086,8 @@ def vlines(self, x, ymin, ymax, colors=None, linestyles='solid',
 
         Other Parameters
         ----------------
+        data : indexable object, optional
+            # data_parameter_message
```

Comment:
This does seem easier to me.  As-is it looks like a comment rather than something that will get magically filled it. Maybe:
```suggestion
            # _data_parameter_placeholder
```

I guess its not possible to just use our usual doc string interpolation here?

Reply:
Indeed, the `#` does not make much sense. I've gone with `DATA_PARAMETER_PLACEHOLDER`, which should reasonably look out-of-context to indicate it's not literally part of the docstring.